Automattic/jetpack
# Google Fonts Module: Update the list of fonts to include additional fonts.

Fixes [#62411](https://github.com/Automattic/wp-calypso/issues/62411)

#### Changes proposed in this Pull Request:

* Updating the list of google fonts to include the additional fonts: IBM Plex Sans + Mono, Jost, Literata, Newsreader, and Texturina

p9Jlb4-44k-p2 for further context.

#### Jetpack product discussion




#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






_Ensure these test ared done for the added fonts:` IBM Plex Sans, IBM Plex  Mono, Jost, Literata, Newsreader, and Texturina`_

* On a site with the most recent version of Gutenberg active, enable the Google Fonts module (/wp-admin/admin.php?page=jetpack_modules).
* With a block theme enabled, to the Site editor.
* Go to Global styles and choose a Google Font from the Font family settings for all Text.
* Go to the Blocks settings in Global styles and choose a different Google Font for the font family setting for all blocks of that type, like Buttons (makes sure you have an instance of that block on the page).
* Select an individual block's settings and set yet another Google font for that individual block (like a List block... not all blocks have Font family settings, it partly depends on the theme).
* Now load the the page you were editing in the front end... make sure all of the fonts you selected are loaded. I do this by viewing the Network tab in browser tools and filtering for Fonts, or the filtering for the fonts.gstatic.com domain.
